olp-cpp-sdk-core/src/http/winhttp/NetworkWinHttp.cpp

```diff
@@ -977,7 +1002,17 @@ NetworkWinHttp::RequestData::RequestData(
       resumed(false),
       ignore_data(request.GetVerb() == NetworkRequest::HttpVerb::HEAD),
       no_compression(false),
-      uncompress(false) {}
+      uncompress(false),
+      in_use(false) {}
+
+NetworkWinHttp::RequestData::RequestData()
+    : http_request(NULL),
+      request_id(-1),
```

Comment:
There are defines for this -> RequestIdConstants::RequestIdInvalid from NetworkTypes.h.
Please use that to mark an invalid RequestId